packages/fluentui/react-northstar/src/components/Tree/TreeTitle.tsx

```diff
@@ -101,21 +142,51 @@ const TreeTitle: React.FC<WithAsProp<TreeTitleProps>> & FluentComponentStaticPro
 
   const ElementType = getElementType(props);
   const unhandledProps = useUnhandledProps(TreeTitle.handledProps, props);
-
   const handleClick = e => {
     _.invoke(props, 'onClick', e, props);
   };
 
+  const selectIndicator = Box.create(
+    customSelectIndicator
+      ? {
+          selected,
+          selectable: selectable && !hasSubtree,
+          selectableParent,
+          expanded,
+        }
+      : selectionIndicator,
+    {
+      defaultProps: () => ({
+        as: customSelectIndicator || 'span',
```

Comment:
```suggestion
        as: 'span',
```

Reply:
If we do that, we would never render the custom checkbox?